Tutorium03/sections/Vektorraeume.tex

```diff
@@ -0,0 +1,140 @@
+                                                 v_1 & v_2 & \dots & v_n
+        \end{pmatrix}$
+    \end{itemize}
+\end{frame}
+
+\begin{frame}
+    \frametitle{Mit Vektoren rechnen}
+    Seien v= $(v_1, v_2, \dots, v_n)$ und w= $(w_1, w_2, \dots, w_n)$ zwei $n$-dimensionale Vektoren.
+    \begin{itemize}
+        \item Addition: $v + w = (v_1 + w_1, v_2 + w_2, \dots, v_n + w_n)$
+        \item Subtraktion: $v - w = (v_1 - w_1, v_2 - w_2, \dots, v_n - w_n)$
+        \item Skalarmultiplikation: $c \cdot v = (c \cdot v_1, c \cdot v_2, \dots, c \cdot v_n)$
+        \item Länge: $|v| = \sqrt{v_1^2 + v_2^2 + \dots + v_n^2}$
+        \item Normalisierung: $\hat{v} = \frac{v}{|v|}$
+        \item Skalarprodukt: $v \cdot w = v_1 \cdot w_1 + v_2 \cdot w_2 + \dots + v_n \cdot w_n$
+    \end{itemize}
+\end{frame}
+
+\begin{frame}
+    \frametitle{Vektoren Uebung}
+    Seien $v = (-2, 3, -6)$ und $w = (4, 5, 6)$ zwei $n$-dimensionale Vektoren.
+    Berechnen Sie:
+    \begin{enumerate}
+        \item $v + w$.
+        \item $v - w$.
+        \item $2 \cdot v$.
+        \item $|v|$.
+        \item $\hat{v}$.
+        \item $v \cdot w$.
+    \end{enumerate}
+\end{frame}
+
+\begin{frame}
+    \frametitle{Vektoren Uebung Loesung}
+    Seien $v = (-2, 3, -6)$ und $w = (4, 5, 6)$ zwei $n$-dimensionale Vektoren.
+    \begin{enumerate}
+        \item $v + w = (-2 + 4, 3 + 5, -6 + 6) = (2, 8, 0)$.
+        \item $v - w = (-2 - 4, 3 - 5, -6 - 6) = (-6, -2, -12)$.
+        \item $2 \cdot v = (2 \cdot -2, 2 \cdot 3, 2 \cdot -6) = (-4, 6, -12)$.
+        \item $|v| = \sqrt{(-2)^2 + 3^2 + (-6)^2} = \sqrt{4 + 9 + 36} = \sqrt{49} = 7$.
+        \item $\hat{v} = \frac{v}{|v|} = \frac{(-2, 3, -6)}{7} = \frac{(-2, 3, -6)}{7} = (-\frac{2}{7}, \frac{3}{7}, -\frac{6}{7})$.
+        \item $v \cdot w = (-2 \cdot 4) + (3 \cdot 5) + (-6 \cdot 6) = -8 + 15 + -36 = -43$.
+    \end{enumerate}
+\end{frame}
+
+\begin{frame}
+    \frametitle{Besondere Vektoren}
+    \begin{itemize}
+        \item Nullvektor: $v = (0, 0, \dots, 0)$
+        \item Einheitsvektor: $|v| = 1$
+        \item Vektor der Standardbasis: $e_i = (0, 0, \dots, 1, \dots, 0)$
+    \end{itemize}
+\end{frame}
+
+\subsection{Exkurs}
+\begin{frame}
+    \frametitle{Exkurs: Mathematische Definition Vektorräume}
+    \begin{itemize}
+        \item Gruppe
+        \begin{itemize}
+            \item $(G,*)$, Menge $G$, Verknüpfung $*$
+            \item Assozsiativität
+            \item Existenz eines neutrales Elements
+            \item Existenz eines inversen Elements
+        \end{itemize}
+        \item Ring
+        \begin{itemize}
+            \item $(R,+,*)$ mit $R$ Menge, $+$ Verknüpfung, $*$ Verknüpfung
+            \item $(R,+)$ ist abelsche Gruppe(Gruppe mit Kommutativgesetz)
+            \item $(R,*)$ ist Halbgruppe(nur Assoziativität)
+            \item Distributiv 1: $a * (b + c) = a * b + a * c$
+            \item Distributiv 2: $(a + b) * c = a * c + b * c$
+        \end{itemize}
+        \item Körper
```

Comment:
At least in my compilation and pdf viewer everything below line93 is cutoff and not visible

Reply:
you're right. fixed this by extracting Körper to its own slide